JuiceMaker/JuiceMaker/Controller/ViewController.swift

```diff
@@ -10,6 +10,12 @@ class ViewController: UIViewController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
+        do {
+            try JuiceMaker().takeOrder(order: .strawberryBananaJuice)
+        }
+        catch {
+            
+        }
```

Comment:
```suggestion
        catch {
            print(error)
        }
```

이렇게 해주면 어떤 에러로 인해서 쥬스를 만들 수 없는지 나타낼 수 있을 것 같습니다.